openfe/tests/setup/test_network_planning.py

```diff
@@ -90,6 +90,47 @@ def test_radial_network_failure(atom_mapping_basic_test_files):
         )
 
 
+@pytest.mark.parametrize('with_scorer', [True, False])
+@pytest.mark.parametrize('extra_mapper', [True, False])
```

Comment:
Whilst I'm a man that loves his parametrize, I do wonder if this might be easier to read if it was 3 tests 😅

Reply:
this is probably fine, it's one test doing the "same thing"

Reply:
Was in a hurry to get to some evening plans. 😆 But I'm still happy to leave as-is.